gnomad_qc/v4/resources/release.py

```diff
@@ -19,6 +19,46 @@
     RELEASES,
 )
 
+FREQUENCY_README = """
```

Comment:
@ch-kr Added this but welcome your wordsmithing

Reply:
One note, when this is hl.eval'd, it looks .....bad because eval just translates special characters to have escape sequences...I couldnt find a way around this, it will be on the user to parse

Reply:
running
```
print(hl.eval(ht.readme), sep=os.linesep)
```
helped! it's still a bit ugly but no longer has `\n`s everywhere